### ISSUE TITLE
mDNS

### ISSUE THREAD

**Issue by macsnoeren** [AvansETI/SmartMeter_DIY#5]
I do not understand the following line of code. The mDNS is also not working for me (DIY-EMON_V14.local). I also do not understand why a service is added when the mDNS was not able to start. I also do not understand the service on port 10000. What is that? See it on line 350.
```
  if( !MDNS.begin("DIY-EMON_V14") ) {
  } else {
    MDNS.addService("diy_emon_v14", "tcp", 10000);
  }
```
I would expect the following:
```
  if ( MDNS.begin("diy_smartmeter") ) { 
    MDNS.addService("http", "tcp", 10000);
    Serial.println("\nmDNS: Started");
  } else {
    Serial.println("\nmDNS: Error");
  }
```

**Comment by dkroeske** [comment]
Is this an update on the mdns library? It need some testing, dit you test it?

**Comment by macsnoeren** [comment]
No, this is your initial code. I am busy with some feature and this came across. I cannot figure out what your code is about for mDNS. Maybe you still know ;).

**Comment by macsnoeren** [comment]
I have resolved the problem. MDNS.update() was not called in the loop() to process the functionality.
